.github/workflows/pr.yaml

```diff
@@ -3,8 +3,7 @@ name: Pull Request
 on:
   # Dangerious without Member Check setup!
   pull_request_target:
-    branches:
-      - dev
+    branches: [dev, master]
```

Comment:
CI should run on master too.  See [this discussion](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3822#discussion_r1934828572).